### PR TITLE
Polish import export

### PR DESCRIPTION
**Pull request by michael-wirth**

### CODE REVIEW

File: app-ws-avam-sink/build.gradle [hunk 1]
```diff
@@ -60,7 +35,7 @@ task registerApps(type: HttpTask) {
         request.contentType = 'application/x-www-form-urlencoded'
     }
     post {
-        request.uri.path = '/apps/sink/ws-avam'
+        request.uri.path = '/apps/sink/wsavam'
```
Comment (birom) [inline app-ws-avam-sink/build.gradle]: Why do we change the path. Is there some kind of convention? 
Please also change the readme file.

File: app-ws-avam-source/build.gradle [hunk 2]
```diff
@@ -38,7 +38,7 @@ task registerApps(type: HttpTask) {
         request.contentType = 'application/x-www-form-urlencoded'
     }
     post {
-        request.uri.path = '/apps/source/ws-avam'
+        request.uri.path = '/apps/source/wsavam'
```
Comment (birom) [inline app-ws-avam-source/build.gradle]: Please also change the readme file.

File: app-x28-job-ad-export-task/build.gradle [hunk 3]
```diff
@@ -36,7 +36,7 @@ task registerApps(type: HttpTask) {
         request.contentType = 'application/x-www-form-urlencoded'
     }
     post {
-        request.uri.path = '/apps/task/x28-job-ad-export'
+        request.uri.path = '/apps/task/x28jobadexport'
```
Comment (birom) [inline app-x28-job-ad-export-task/build.gradle]: Please also change the readme file.

File: app-ws-avam-sink/build.gradle (hunk 1 above)
Comment (michael-wirth) [reply app-ws-avam-sink/build.gradle]: the app name doesn't allow - dashes anymore

File: app-x28-job-ad-import-task/build.gradle [hunk 4]
```diff
@@ -47,7 +47,7 @@ task registerApps(type: HttpTask) {
         request.contentType = 'application/x-www-form-urlencoded'
     }
     post {
-        request.uri.path = '/apps/task/x28-job-ad-import'
+        request.uri.path = '/apps/task/x28jobadimport'
```
Comment (birom) [inline app-x28-job-ad-import-task/build.gradle]: Please also change the readme file.

File: app-ws-avam-source/build.gradle (hunk 2 above)
Comment (michael-wirth) [reply app-ws-avam-source/build.gradle]: done

File: app-x28-job-ad-export-task/build.gradle (hunk 3 above)
Comment (michael-wirth) [reply app-x28-job-ad-export-task/build.gradle]: done

File: app-x28-job-ad-import-task/build.gradle (hunk 4 above)
Comment (michael-wirth) [reply app-x28-job-ad-import-task/build.gradle]: done

**Pull request merged**